RFS/build-preloaded-source-image.gradle

```diff
@@ -64,14 +63,14 @@ def sourceContainerCommitTask = task commitSourceContainer() {
         def containerId = createInitialElasticsearchContainer.getContainerId().get()
 
         client.commitCmd(containerId)
-                .withRepository("elasticsearch_rfs_source")
+                .withRepository("migrations/elasticsearch_rfs_source")
                 .withTag("latest")
                 .exec()
     }
 }
 
 task removeClientContainer(type: DockerRemoveContainer) {
-    dependsOn waitClientContainer
+    dependsOn commitSourceContainer
```

Comment:
why would you need to wait for the source to be committed?  Shouldn't the client finishing be independent of when the sync happens?

Reply:
Good question.  We probably don't need to do this, but it made the sequence of operations clearer in my mind when I was trying to figure out why the `commitSourceContainer` task didn't appear to be firing.  In reality the commit task was firing, it just didn't report anything to stdout.